map/src/App.tsx

```diff
@@ -52,9 +54,17 @@ class App extends React.Component<Props, State> {
       fullScreen: false,
       resultsMode: 'open',
       updateResultsOnNextClustering: false,
+      translate: {},
     };
   }
 
+  componentDidUpdate(prevProps: Props, prevState: State): void {
+    const { translate } = this.state;
+    if (prevState.translate !== translate) {
+      window.location.reload(false);
```

Comment:
we can avoid reloading the page when the language updates by handling state a little differently

Reply:
(for future reference, this could also have been put in the `setTranslate` function directly)